.CI/create_feedstocks.py

```diff
@@ -254,10 +252,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
```

Comment:
Appears our `git` is not new enough to support the `--autostash` flag. So we are enabling autostash with all rebases, which seems to solve the problem in PR ( https://github.com/conda-forge/staged-recipes/pull/1469 ).